src/main/java/com/example/naitei19javaecommerce/controller/user/HomeController.java

```diff
@@ -7,6 +7,6 @@
 public class HomeController {
     @GetMapping("/home")
     public String Getbase() {
-        return "layout/user/base-layout";
+        return "user/homepage/home-page";
```

Comment:
```suggestion
        return "user/static-pages/home-page";
```
sửa lại tên folder cho nó hợp lý hơn emm nhé

Reply:
em thắc mắc sao lại static-pages vậy c , tại nó là trang home ấy